README.md

```diff
@@ -515,6 +515,30 @@ Returns the publicly available settings for this gotrue instance.
 }
 ```
 
+### **POST, PUT /admin/users/<user_id>**
+
+Creates (POST) or Updates (PUT) the user based on the `user_id` specified. The `ban_duration` field accepts the following time units: "ns", "us", "ms", "s", "m", "h". 
+
+```js
+headers:
+{
+  "Authorization": "Bearer eyJhbGciOiJI...M3A90LCkxxtX9oNP9KZO" // admin role required
+}
+
+body:
+{
+  "role": "test-user",
+  "email": "email@example.com",
+  "phone": "12345678",
+  "password": "secret", // only if type = signup
+  "email_confirm": true,
+  "phone_confirm": true,
+  "user_metadata": {},
+  "app_metadata": {},
+  "ban_duration": "24h"
```

Comment:
sorry this is pedantic but is there a link to the format spec - e.g. can I do `24h 32m 6s` or `24h 3s` or `1y3m2s` (with no spaces) ?

Reply:
Yup, this is the format spec it follows
https://pkg.go.dev/time#ParseDuration